packages/ui-library/src/components/date-time-picker/RuiDateTimePickerMenu.vue

```diff
@@ -74,18 +74,16 @@ function setNow(): void {
             />
           </RuiButton>
         </div>
-        <div class="flex flex-col">
-          <RuiAutoComplete
-            v-model="selectedTimezone"
-            hide-details
-            label="Timezone"
-            class="pt-0 pr-0"
-            variant="outlined"
-            :options="timezones"
-          />
-          <slot />
-        </div>
+        <RuiAutoComplete
+          v-model="selectedTimezone"
+          hide-details
+          label="Timezone"
+          class="pt-0 pr-0"
+          variant="outlined"
+          :options="timezones"
+        />
       </div>
+      <slot />
```

Comment:
Where would this put the slot on the UI now? before it was supposed to go under the timezone.

Reply:
it's under the div that wrapped both <button to set to now | the timezone selector >